api/admincontract.go

```diff
@@ -29,6 +29,8 @@ func NewAdminContractService(runner *Runner) *AdminContractService {
 	}
 	allowedMethods["contract.registerNode"] = true
 	allowedMethods["contract.getNodeRef"] = true
+	allowedMethods["schema.get"] = true
```

Comment:
`schema.get` is not a contract method, so you don't need to add it here

Reply:
ok